docs/release-notes/8.12.asciidoc

```diff
@@ -1,6 +1,28 @@
 [[release-notes-header-8.12.0]]
 == 8.12
 
+[discrete]
+[[release-notes-8.12.1]]
+=== 8.12.1
+
+[discrete]
+[[enhancements-8.12.1]]
+==== Enhancements
+* New Defend Advanced Policy Options for 8.12.1 ({pull}175486[#175486]).
+
+[discrete]
+[[bug-fixes-8.12.1]]
+==== Bug fixes
+* Fixes misaligned elements in the top navigation bar ({pull}175516[#175516]).
+* Fixes a bug with the agent select field in Osquery bug with `-` ({pull}175134[#175134]).
```

Comment:
@tomsonpl I could use your help with the summary for https://github.com/elastic/kibana/pull/175134 🙏🏼 Does your PR fix a bug that affects the list of results returned when users include a dash and wildcard character in their agent search query?

Reply:
Hey :) Actually it's just about `-` .  Meaning that if we have a 'jammy-1234' agent. The search works if we search for 'jammy' , but if we go further with 'jammy-' it will not return the results. 

Wildcard was mentioned just because it is how the query looks like in the code. 
Hope this helps :) Thanks @nastasha-solomon

Reply:
Ah, ok thanks for the context. How about this:

```suggestion
* Fixes a bug that affected search results when you entered an agent name that included a dash (`-`) ({pull}175134[#175134]).
```